### PR TITLE
Dev handlers auth impl

### PR DESCRIPTION
**Pull request by VitaliiFriber**
Please check over. Do i understand correctly what we should make?

### CODE REVIEW

File: services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * Created by friber on 26.02.17.
+ */
+
+@Component
+public class AuthLoginHandler extends AbstractRequestHandler<LoginRequest, LoginResponse>
+        implements AuthorizationRequestHandler, RouterHandler {
+    private String handlerName = "login";
+
+    public String getRouteKey(Request<?> msg) {
```
Comment (Dominys) [inline services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java]: @VitaliiFriber Why we need this here ?

File: services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by friber on 26.02.17.
+ */
+
+@Component
+public class AuthLoginHandler extends AbstractRequestHandler<LoginRequest, LoginResponse>
+        implements AuthorizationRequestHandler, RouterHandler {
+    private String handlerName = "login";
```
Comment (Dominys) [inline services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java]: @VitaliiFriber If you create constant - than make it static final.
Better to move constants to separate class.

File: services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthRegisterHandler.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    public String getName() {
+        return handlerName;
+    }
+
+    public Response<RegisterResponse> handle(Request<?> request) {
+        return new Response<RegisterResponse>();
+    }
+
+    public String getRouteKey(Request<?> msg) {
```
Comment (Dominys) [inline services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthRegisterHandler.java]: @VitaliiFriber ?

File: services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java (hunk 2 above)
Comment (VitaliiFriber) [inline services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java]: Do it in class? in enum? or return hardcoded string on the method?

File: services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java (hunk 1 above)
Comment (VitaliiFriber) [inline services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java]: Sorry,mistakes.

File: services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java (hunk 2 above)
Comment (Dominys) [inline services/authorization/authorization-impl/src/main/java/com/softgroup/authorithation/impl/handler/AuthLoginHandler.java]: @VitaliiFriber Make it final static field. Or better move all commands to some AuthorizationCommands class

**Pull request closed without merging**